test/extended/crc/cmd/cmd.go

```diff
@@ -97,11 +97,23 @@ func (c Command) env() []string {
 	return env
 }
 
-func envVariable(key, value string) string {
+func EnvVariable(key, value string) string {
+	var prefix = ""
+	var suffix = ""
 	if runtime.GOOS == "windows" {
-		return fmt.Sprintf("$env:%s=%s;", key, value)
+		prefix = "$env:"
+		suffix = ";"
+		switch value {
+		case "true":
+			value = "$true"
+			break
+		case "false":
+			value = "$false"
+			break
+		default:
+		}
```

Comment:
Do you think we can replace this switch statement like this?
```suggestion
                value = fmt.Sprintf("$%s", value)
```

Reply:
maybe the function should be called `SetBoolEnvVariable` then, because we want to append the `$` only in case of bool values

Reply:
This cannot be applied as this deals too with the case of env var not being a bool, in which case, value is keep unmodified:

```
FOO=bar
$env:FOO=bar
```

```
BAR=true
$env:BAR=$true
```

this is problematic if we pretend to set an env var with text "true"

Reply:
okay, I'm sorry I didn't know about this use case. 

marking this comment as resolved.